docs/src/api.md

```diff
@@ -285,9 +285,12 @@ Pathfinding.HeightMap
 Building a custom metric is straightforward, if the provided ones do not suit your purpose.
 See the [Developer Docs](@ref) for details.
 
-## AgentsIO
+## (De)Serialization
```

Comment:
I think this should be named `Save/Load` instead. E.g. the CSV functionality isn't really serialization. Also, I don't even think that the average Agents.jl user would know what the world serialization means.